kaggle/api/kaggle_api_extended.py

```diff
@@ -1316,24 +1349,53 @@ def dataset_metadata_update(self, dataset, path):
     meta_file = self.get_dataset_metadata_file(effective_path)
     with open(meta_file, 'r') as f:
       metadata = json.load(f)
-      updateSettingsRequest = DatasetUpdateSettingsRequest(
-          title=metadata['title'],
-          subtitle=metadata['subtitle'],
-          description=metadata['description'],
-          is_private=metadata['isPrivate'],
-          licenses=[License(name=l['name']) for l in metadata['licenses']],
-          keywords=metadata['keywords'],
-          collaborators=[
-              Collaborator(username=c['username'], role=c['role'])
-              for c in metadata['collaborators']
-          ],
-          data=metadata['data'])
-      result = self.process_response(
-          self.metadata_post_with_http_info(owner_slug, dataset_slug,
-                                            updateSettingsRequest))
-      if (len(result['errors']) > 0):
-        [print(e['message']) for e in result['errors']]
-        exit(1)
+      with self.build_kaggle_client() as kaggle:

```

Comment:
Is this intentional?  The title of this change is `Convert arg strings to enum values`.  This looks like part of the Datasets Kapigen conversion.

Reply:
I forgot to do this one when I worked on datasets.